Source/UI/CommonUI/UIStyle/CKOUIStyle+UIColor.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+
+extension CKOUIStyle {
```

Comment:
We are not using any of these so I am unsure why they're here. 

Also you've added this to the SDK, not the sample app, which means we ship this to clients?

Reply:
If "nothing new", then code changes should take place (some code removed, some added). But this is all just new code.

I also just remembered that @deepesh-vasthimal-cko was not keen on CKO to be shown externally, being more of an internal abbreviation. This code is open sourced so shouldn't show that.